aten/src/ATen/native/cuda/jit_utils.h

```diff
@@ -101,4 +102,21 @@ template <> inline std::string typeName<at::BFloat16>(){
     return "at::BFloat16";
 }
 
+
+#define TYPE_NAME_CASE(ctype, scalartype)                    \
+  case ScalarType::scalartype:  return std::string(#ctype);
+inline std::string typeName(ScalarType t) {
+    switch (t) {
+        // TODO: check why bool is not included
```

Comment:
Why?

Reply:
No particular reason, we didn't have jiterated functions that had scalar_t bool?